engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -95,6 +95,46 @@ paths:
           $ref: '#/responses/InvalidSubmission'
         '500':
           $ref: '#/responses/ServerError'
+  '/api/workflows/{version}/describe':
+    post:
+      operationId: describe
+      summary: Describe a workflow description
+      description: Analyze a workflow description for validity, input requirements and other metadata
+      consumes:
+      - multipart/form-data
+      parameters:
+      - $ref: '#/parameters/versionParam'
+      - name: workflowSource
+        description: The workflow source file to submit for execution. Either workflow source or workflow url is required.
+        required: false
+        type: file
+        in: formData
+      - name: workflowUrl
+        description: URL which points to the workflow. Either workflow source or workflow url is required.
+        required: false
+        type: string
+        in: formData
```

Comment:
it looks like I can send either a file or a URL as input. Can I send raw WDL (or CWL, etc)? That's the Workbench use case.

Reply:
I think that should just be the file case

Reply:
as a calling client, there's a structural difference between building a legal/valid `multipart/form-data` payload (more complex) and simply specifying a text clob (simpler). I'd really like to not jump through the extra hoops here. BUT ... after looking, I realize this is parallel to what Cromwell supports for submitting workflows. I prefer the consistency between the two APIs over making one API easier.

Reply:
Yeah, my WIP implementation routes requests through pretty much the same code as submitted workflows

( The WIP implementation is progressing on a separate branch to allow the Swagger spec to be discussed on its own here. )